src/Core/Validation/AdjectiveValidation.fs

```diff
@@ -0,0 +1,38 @@
+﻿module AdjectiveValidation
+
+open Article
+open AdjectiveArticle
+open Comparison
+open ComparativeBuilder
+
+let hasMorphologicalComparatives = 
+    getComparatives 
+    >> Array.filter isMorphologicalComparison
+    >> (not << Seq.isEmpty)
+
+let tryGetAdjective =
+    tryGetContent
+    >> Option.filter (hasChildPart "čeština")
+    >> Option.map (getChildPart "čeština")
+    >> Option.filter (hasChildPart "přídavné jméno")
+    >> Option.map (getChildPart "přídavné jméno")
+
+let hasRequiredInfoComparative =
+    tryGetAdjective
+    >> Option.exists (hasChildPart "stupňování")
+
+let hasRequiredInfoPlural = 
+    tryGetAdjective 
+    >> Option.exists (hasChildPart "skloňování")
+
+let isValidCommon = isPositive
```

Comment:
Not sure what this name means, a bit unclear for me

Reply:
So, some adjectives are valid for the `Plural` exercise, other are valid for the `Comparative` exercise, most are actually valid for both. Thus we have different validation functions and common validation steps are included in the `isValidCommon` function. In this case, there is only one common check but there can be more stuff (consider `isValidCommon` in `NounValidation`).

Reply:
Can it be named `isValid` or `isValidAdjective`? You can keep `isValidCommon` if those don't seem to fit

Reply:
Good idea, did it in the neighboring modules as well